internal/olm/operator/internal/configmap.go

```diff
@@ -120,9 +120,9 @@ func makeObjectFileName(b []byte, names ...string) string {
 	return fileName + "yaml"
 }
 
-// hashContents creates a base32-encoded md5 digest of b's bytes.
+// hashContents creates a sha256 digest of b's bytes.
 func hashContents(b []byte) string {
-	h := md5.New()
+	h := sha256.New()
```

Comment:
For example here, why the md5 would be a risk in this case? Why we should change to `sha256`? Shows that we have no reason to do this change at all. the only reason here is that the CI pass in the gosec check,

Reply:
@camilamacedo86 this is not true... This change was not put in place to satisfy the linter... md5 has been reported as broken and unsuitable since 2010.   sha256 is a good recommended solution. 

https://www.md5online.org/blog/why-md5-is-not-safe/